Library/Formula/antigen.rb

```diff
@@ -0,0 +1,16 @@
+class Antigen < Formula
+  desc "A plugin manager for zsh, inspired by oh-my-zsh and vundle."
+  homepage "http://antigen.sharats.me/"
+  url "https://github.com/zsh-users/antigen/archive/v1.tar.gz"
+  sha256 "6d4bd7b5d7bc3e36a23ac8feb93073b06e1e09b9100eb898f66c2e8c3f4d7847"
+  head "https://github.com/zsh-users/antigen.git"
+
+  def install
+    lib.install "antigen.zsh"
```

Comment:
Install this to `share` or `libexec`: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/27900/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_antigen___strict/